src/aura/STG_CMP_Affl/STG_CMP_AfflHelper.js

```diff
@@ -94,7 +94,21 @@
                 
                 // Re-Disable Add Setting button after successful completion
                 component.find("newAfflMappingBtn").set("v.disabled", true);
+                
             } else if(response.getState() === "ERROR") {
+                var errors = JSON.stringify(response.getError());
+                if (errors.includes('FIELD_INTEGRITY_EXCEPTION')) {                
+                    var tst = component.find("failureToast");
+                    $A.util.removeClass(tst, "slds-hide");
+                    $A.util.addClass(tst, "slds-show");
+                
+                    window.setTimeout(
```

Comment:
The error message looks good now. However, it takes some time to read it and process it. Specially for someone not familiar beforehand with what it says. Let's keep it open and let the user close it. Thanks!

Reply:
Hi @ceiroa Updated based on feedback. Back to you. Thanks!